recipes/flock/meta.yaml

```diff
@@ -0,0 +1,13 @@
+package:
+  name: flock
+  version: "1.0"
+source:
+  git_url: https://github.com/cristhomas/immport-test.git
```

Comment:
It's a github trick. Is the commit hash and the archive url. The ending `tar.gz` can be exchanged to `.zip` of example.